python/ray/serve/schema.py

```diff
@@ -608,7 +608,25 @@ def get_empty_schema_dict() -> Dict:
 
 
 @PublicAPI(stability="alpha")
-class ReplicaDetails(BaseModel, extra=Extra.forbid, frozen=True):
+class ServeActorDetails(BaseModel, frozen=True):
+    node_id: Optional[str] = Field(
```

Comment:
can we also add status here?

Reply:
Do you mean for controller specifically? I'm not really sure how we would track controller status - is it possible to use whatever is used by the dashboard for the plain actor info? If the controller is down, the Serve REST API would fail anyways.

Reply:
can you add the actor status for the controller as the status? The frontend could fetch it, but might as well send it down with the initial response to avoid the back and forth